manifests/example/upload-token.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: upload.cdi.kubevirt.io/v1alpha1
 kind: UploadTokenRequest
 metadata:
-  name: upload-test
+  name: upload-datavolume
```

Comment:
This change breaks the upload-pvc example, as that is expecting upload-test name, please create another upload-dv-token.yaml and leave this one as is.